src/frontmatter.xml

```diff
@@ -35,84 +35,87 @@
-
-
--->
-<!--
-
-        <p>Though there are no specific prerequisites for a course in abstract algebra, students who have had other higher-level courses in mathematics will generally be more prepared than those who have not, because they will possess a bit more mathematical sophistication. Occasionally, we shall assume some basic linear algebra; that is, we shall take for granted an elementary knowledge of matrices and determinants. This should present no great problem, since most students taking a course in abstract algebra have been introduced to matrices and determinants elsewhere in their career, if they have not already taken a sophomore or junior-level course in linear algebra.</p>
-
-        <p>Exercise sections are the heart of any mathematics text. An exercise set appears at the end of each chapter. The nature of the exercises ranges over several categories; computational,  conceptual, and theoretical problems are included. A section presenting hints and solutions to many of the exercises appears at the end of the text. Often in the solutions a proof is only sketched, and it is up to the student to provide the details. The exercises range in difficulty from very easy to very challenging. Many of the more substantial problems require careful thought, so the student should not be discouraged if the solution is not forthcoming after a few minutes of work.</p>
--->
-        <!-- Removed reference to the solutions manual.  TWJ 8/19/2010 -->
-        <!-- A complete solutions manual is available for the instructor's use.  -->
-<!--
-        <p>There are additional exercises or computer projects at the ends of many of the chapters. The computer projects usually require a knowledge of programming. All of these exercises and projects are more substantial in nature and allow the exploration of new results and theory.</p>
--->
-        <!-- % Added Sage blurb, RAB 2011/07/28 -->
-<!--        <p>Sage (<url href="http://sagemath.org">sagemath.org</url>) is a free, open source, software system for advanced mathematics, which is ideal for assisting with a study of abstract algebra. Sage can be used either on your own computer, a local server, or on SageMathCloud (<url href="https://cloud.sagemath.com">https://cloud.sagemath.com</url>). Robert Beezer has written a comprehensive introduction to Sage and a selection of relevant exercises that appear at the end of each chapter, including live Sage cells in the web version of the book.</p>
--->
-        <!-- RAB 2014/08/18 Should move flush right with some vertical spacing -->
-<!--        <attribution>Thomas W. Judson <br />Nacogdoches, Texas 2015</attribution>
-
-    </preface>-->
+    <acknowledgement>
+        <p>Over several terms, Jack Green wrote most of the book's content as a Word/PDF document for use in his face-to-face and online Precalculus courses at Mt. Hood Community College.</p>
```

Comment:
I also changed this above, where "Mt." appears in the <titlepage><author><institution>.